lib/configurators/aws/aws.go

```diff
@@ -358,18 +351,19 @@ To avoid seeing this warning, please provide a region in your AWS config or thro
 	return "us-east-1"
 }
 
+func getSTSClient(cfg aws.Config) *sts.Client {
+	return stsutils.NewFromConfig(cfg, func(o *sts.Options) {
+		o.TracerProvider = smithyoteltracing.Adapt(otel.GetTracerProvider())
+	})
+}
+
 // CheckAndSetDefaults checks and set configuration default values.
 func (c *ConfiguratorConfig) CheckAndSetDefaults() error {
 	ctx := context.Background()
 	if c.ServiceConfig == nil {
 		return trace.BadParameter("config file is required")
 	}
 
-	useFIPSEndpoint := aws.FIPSEndpointStateUnset
-	if modules.GetModules().IsBoringBinary() {
-		useFIPSEndpoint = aws.FIPSEndpointStateEnabled
-	}
-
```

Comment:
I'm not familiar with FIPS, but why do we no longer use the FIPS endpoint?

Reply:
We still do, it's just in the `awsconfig` package now and gets pulled in with `awsconfig.GetConfig()`: https://github.com/gravitational/teleport/blob/99584033e14cfa3c420c63794d4aa5fc2fd053c6/lib/cloud/awsconfig/awsconfig.go#L279-L281